TSG/Networking/Top-Of-Rack-Switch/Troubleshooting-TOR-Overview-Guidance.md

```diff
@@ -0,0 +1,538 @@
+![Switchless with 2 ToRs](images/AzureLocalPhysicalNetworkDiagram_Switchless.png)
+
+### Comparison Summary: Fully-Converged vs Switched vs Switchless
+
+## Network Design Comparison
+
+| Design Type         | Host NIC Traffic                                                   |  VLAN on ToRs                       | Typical Use Case                                      |
+|---------------------|---------------------------------------------------------------------|--------------------------------------------------------|--------------------------------------------------------|
+| **Fully Converged** | 2 NICs per host. All traffic (M + C + S) over both NICs | Host-facing ports are configured as trunks carrying M, C, and S VLANs — all VLANs must be allowed on both ToRs | Simple, scalable design with RDMA and minimal cabling |
+| **Switched**        | 4 NICs per host. 2 for M + C, 2 for S                | Host-facing ports are configured as trunks: M and C VLANs are present on both ToRs; S1 VLAN is only on ToR1, and S2 VLAN is only on ToR2 | For dedicated storage performance and traffic separation |
+| **Switchless**      | 2 NICs per host to switch (M + C) + (N−1) direct host-to-host NICs for S | Trunk to host with M, C VLANs only; no S VLANs on ToRs | Low-cost, small-site or edge deployments without storage switches |
+
+
+> [!IMPORTANT]  
+> Storage VLANs are preferably configured as **Layer 2 (L2) VLANs without IP subnets** to simplify the overall network design. Since storage traffic is **tagged by Azure Local hosts**, make sure these VLANs are always configured as **tagged VLANs (trunk ports)** on the **ToR (Top of Rack)** switches.
+
+
+---
+
+## 3. Physical Network Hardware & Configuration Requirements
+
+### Network Switch Requirements
+
+Azure Local requires specific capabilities from physical network switches to ensure performance, compatibility, and integration with RDMA, SET, and VLAN-based traffic separation.
+
+You can find the official list of switch requirements here:  
+🔗 [Azure Local – Physical Network Switch Requirements](https://learn.microsoft.com/en-us/azure/azure-local/concepts/physical-network-requirements?view=azloc-2506&tabs=overview%2C24H2reqs#network-switch-requirements)
+
+If you're an **OEM** and wish to qualify your switches for Azure Local deployments, Microsoft provides an open-source validation tool:  
+🛠️ [AzureLocal-Network-Switch-Validation](https://github.com/microsoft/AzureLocal-Network-Switch-Validation)
+
+
+### Network Configuration Requirements
+
+The physical network must be configured in alignment with Azure Local’s host networking requirements, and details and official configuration guidance are available here:  
+🔗 [Azure Local – Host Network Requirements](https://learn.microsoft.com/en-us/azure/azure-local/concepts/host-network-requirements?view=azloc-2506)
```

Comment:
Can you remove the ?view=azloc-2506, this way this link always diretcs to the latest release